src/main/java/seedu/address/logic/commands/exceptions/CommandException.java

```diff
@@ -1,5 +1,7 @@
 package seedu.address.logic.commands.exceptions;
 
+import seedu.address.logic.commands.Command;
```

Comment:
Doesnt look like Command is used here. Why import it?

Reply:
It's a really minor thing, on line 6 it {@link Command} in the Javadoc, so if we dont import it, it wont display properly